src/components/DatePicker/index.js

```diff
@@ -48,36 +53,27 @@ function DatePicker({containerStyles, defaultValue, disabled, errorText, inputID
     const styles = useThemeStyles();
     const [selectedDate, setSelectedDate] = useState(value || defaultValue || undefined);
 
-    useEffect(() => {
-        if (selectedDate === value || _.isUndefined(value)) {
-            return;
-        }
-        setSelectedDate(value);
-    }, [selectedDate, value]);
```

Comment:
Removing this useEffect makes the value prop behaves just like defaultValue thus making the DatePicker a non-controllable component (you can't change it's value programmatically). This does not seem to be a problem now so it may not be a blocker but I think we need to at least make it clear that the value prop is only there because it's needed by the FormProvider (and it will not control the datepicker actual value)

cc @luacmartins thoughts on the above ^

Reply:
That seems fine to me